example/ideScripts/utilities.py

```diff
@@ -49,15 +50,38 @@ def printAndQuit(msg):
     sys.exit(1)
 
 
-def fileFolderExists(path):
+def pathExists(path):

```

Comment:
I would separate file/folder path check function from commands on PATH check function. The reason is that normally term _path_ represent a string to a given file/folder, not associated program - it is not very intuitive to have the same function to check path as _pathExists("myProject/myFile.py")_ and _pathExists("python")_. A little easier to understand for future colaborators.

Function name `pathExists()` is better than the old one.

Reply:
I agree, this was mainly done for my ease for detecting the default paths, will fix.